config/pipeline.yaml

```
@@ -777,6 +776,15 @@ jobs:
       - 'kernelci'
       - 'stable-rc'
```

We also need to enable build with `multi_v7_defconfig`:
```suggestion
      - 'stable-rc'
      - 'omap'
```

---

Yes, that's true. Sorry for the noise.